app/src/main/java/com/sdp/movemeet/utility/ActivitiesUpdater.java

```diff
@@ -28,7 +31,7 @@
 
     private static final String TAG = "Activities updater TAG";
 
-    public static final int MAX_NUMBER_ACTIVITIES = 10;
+    public static final int MAX_NUMBER_ACTIVITIES_TO_DISPLAY = 10;
```

Comment:
This value is fixed for the moment but it should vary according to the choice of the user to filter the closest activity, right?

Reply:
Exactly

Reply:
The updater is not optimized yet, so displaying too much activities on the map makes the app crash. That's why I set up a threshold, but in the near future I'll optimize it so that we add all activities to the list. The distance filtering that Mark implemented will then be able to only display closest activities indeed.